### DISCUSSION TITLE
How to connection with webpack

### DISCUSSION THREAD

**Discussion by Seongseokwon** [maxGraph/maxGraph#161] [Q&A] [answered]
i using Angular 11 
 i want to angular custom webpack with maxgraph 
 how do i do ?

**Comment by tbouffard** [comment, marked as answer]
Hi, can you explain what you exactly want to do? Do you want to integrate `maxGraph` in an Angular 11 application? 
Have you already try this yourself and get some errors? Can you share that with us here please?

Please also check https://github.com/maxGraph/maxgraph-integration-examples/issues/20

**Comment by Seongseokwon** [comment]
I solved it.
But I have a new question,
Dose mathUtils no longer support convertPoint ??

**Reply by Seongseokwon** [reply]
Is it no longer supported because it can be solved using the offset value of the event?

**Reply by f-wrobel** [reply]
I think `convertPoint` is in `styleUtils`, not `mathUtils`.

**Reply by tbouffard** [reply]
yes, exactly.
Source: https://github.com/maxGraph/maxGraph/blob/v0.1.0/packages/core/src/util/styleUtils.ts#L295-L316

**Reply by tbouffard** [reply]
Thanks @f-wrobel for having created #164 about the `convertPoint` issue 🥇
